packages/react/src/lib/felaRenderer.tsx

```diff
@@ -24,6 +31,7 @@ const createRendererConfig = (options: any = {}) => ({
     felaPluginFallbackValue(),
     ...(options.isRtl ? [rtl()] : []),
   ],
+  filterClassName,
```

Comment:
https://github.com/rofrischmann/fela/blob/v10.2.1/packages/fela/src/generateClassName.js#L25

Reply:
Just as I see it here: you can now use the rtl plugin with a theme.direction property to trigger rtl transformation (check the plugin docs) that way you dont have to conditionally add the plugin

Reply:
Yep, seen these changes. Going to visit this place in a separate PR 👍